softlayer/cpi.yml

```diff
@@ -0,0 +1,98 @@
+---
+- type: replace
+  path: /releases/-
+  value:
+    name: bosh-softlayer-cpi
+    version: 10
+    url: https://bosh.io/d/github.com/cloudfoundry/bosh-softlayer-cpi-release?v=10
+    sha1: f0209e5565b64beccffa4482945c865890b5ca71
+
+- type: replace
+  path: /resource_pools/name=vms/stemcell?
+  value:
+    url: https://bosh.io/d/stemcells/bosh-softlayer-xen-ubuntu-trusty-go_agent?v=3363.12.1
+    sha1: c5350a9e0b03a4f9af2b89525be620cb77bc6a4f
+
+- type: replace
+  path: /resource_pools/name=vms/env
+  value:
+    bosh:
+      keep_root_password: true
+
+# Configure sizes
+- type: replace
+  path: /resource_pools/name=vms/cloud_properties?
+  value:
+    domain: ((sl_vm_domain))
+    vmNamePrefix: ((sl_vm_name_prefix))
+    ephemeralDiskSize: 100
+    startCpus: 4
+    maxMemory: 8192
+    deployedByBoshcli: true
```

Comment:
Actually, it's used to get ride of `bosh_ip` from each resource pools (in manifest v1), for deploying bosh director, there needs to update /etc/hosts for supporting dynamic IP. Below is the code where to use it.

https://github.com/cloudfoundry/bosh-softlayer-cpi-release/blob/master/src/bosh-softlayer-cpi/softlayer/vm/softlayer_virtual_guest_creator.go#L103-L130